akka-http2-support/src/test/scala/akka/http/scaladsl/Http2ServerTest.scala

```diff
@@ -67,7 +67,8 @@ object Http2ServerTest extends App {
       } yield (binding1, binding2)
 
     Await.result(bindings, 1.second) // throws if binding fails
-    println("Server online at http://localhost:9001")
+    println("Server (HTTP/1.1) online at http://localhost:9000")
+    println(Console.BOLD + "Server (HTTP/2) online at http://localhost:9001" + Console.RESET)
```

Comment:
We can try to change the example above to use the common `Http()` api and set the flag in code.